notebooks/xvcfmerge-array-input.py

```diff
@@ -0,0 +1,126 @@
+# publish to: "terra-notebook-utils-tests" "test"
+import os
+import callysto
+
+with callysto.Cell("markdown"):
+    """
+    # xvcfmerge with `Array[String]` input
+
+    This notebook demonstrates the creation of Terra data tables as input to xvcfmerge workflows, referencing input VCFs with Array[String].
+
+    This input style is in development. xvcfmerge workflows should use version "xbrianh-input-format"
+
+    # Cohort VCF Merge
+
+    The TopMED data indexed by BioData Catalyst has been jointly called for each Freeze. TOPMed researchers not part of the TOPMed Constorium receive jointly called VCFs subsetted by project and consent code. If users have access to multiple projects and consent codes, they may wish to re-combine these VCFs. This notebook demonstrates merge tooling for jointly called VCFs.
```

Comment:
TopMED multi-sample VCF files indexed by BioData Catalyst have been jointly called for each "Freeze", or release of genomic data. These files are accessible in the "Reference File" node of the Gen3 graph.
TOPMed researchers not part of the TOPMed Consortium receive jointly called VCFs subsetted by project and consent code. If users have access to multiple projects and consent codes, they may wish to re-combine these VCFs to form a synthetic cohort. The Merge & Subsample VCFs notebook demonstrates tooling to merge and subsample jointly called VCFs.

Reply:
fixed